crates/testing/src/test_runner.rs

```diff
@@ -110,10 +123,15 @@ where
     I: NodeImplementation<TYPES, QuorumNetwork = N, CommitteeNetwork = N>,
 {
     /// excecute test
+    ///
+    /// # Arguments
+    /// *  `skip_late` - Whether to skip initializing nodes that will start late, which will catch
+    /// up later with `HotShotInitializer::from_reload` in the spinning task.
+    ///
     /// # Panics
     /// if the test fails
     #[allow(clippy::too_many_lines)]
-    pub async fn run_test(mut self) {
+    pub async fn run_test(mut self, skip_late: bool) {
```

Comment:
I think this should be a parameter in the test metadata instead of an argument here